### DISCUSSION TITLE
i lost access to the email linked to my github hence i can not receive the code sent to it.

### DISCUSSION THREAD

**Discussion by 0xMimisco** [community/community#128618] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

please how do i change an email linked to my github account, i have lost access to the one linked to the email.. please help

**Comment by IshworTM** [comment]
Simply, make a ticket and describe your issue here:
https://support.github.com/tickets/personal/0

**Comment by LiteBrite82** [comment]
Hi @0xMimisco ! 

The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by LiteBrite82** [reply]
Also, see our [documentation](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/adding-an-email-address-to-your-github-account)

**Reply by 0xMimisco** [reply]
cant access the email again.. so there is no way to do it
